lib/CarSpeed/CarSpeed.cpp

```diff
@@ -65,10 +69,21 @@ void CarSpeed::task() {
 
   // polling loop
   while (1) {
-    // update pid controller
+
+    // read target speed
     input_value = get_current_speed();
+
+    // update pid controller
     pid.Compute();
-    dac.set_pot(output_setpoint, DAC::pot_chan::POT_CHAN0); // TOOD: check that the value is in range
+
+    // set acceleration & deceleration // TOOD: check that the value is in range
+    if (output_setpoint > 0) {
+      dac.set_pot(output_setpoint, DAC::pot_chan::POT_CHAN0); // acceleration
+      dac.set_pot(0, DAC::pot_chan::POT_CHAN1);               // deceleration
+    } else {
+      dac.set_pot(0, DAC::pot_chan::POT_CHAN0);                // acceleration
+      dac.set_pot(-output_setpoint, DAC::pot_chan::POT_CHAN1); // deceleration
+    }
 
```

Comment:
I am not shure if it is right to set the potis here. Eventually it is better to put it into carControl.